spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/liquibase/LiquibaseDataSource.java

```diff
@@ -34,7 +34,15 @@
 @Target({ ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER, ElementType.TYPE, ElementType.ANNOTATION_TYPE })
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
-@Qualifier
+@Qualifier("liquibaseDataSource")
```

Comment:
This one was necessary because the custom qualifier's value is sensitive to the annotation properties.

i.e.:
`@LiquibaseDataSource` does not match `@LiquibaseDataSource(closeDataSourceOnceMigrated = false)`

Reply:
Hopefully there's a nicer way to go about it...